Cargo.toml

```diff
@@ -34,6 +34,8 @@ arrayvec = { version = "0.7.4", features = ["serde"] }
 small-fixed-array = { version = "0.4", features = ["serde"] }
 bool_to_bitflags = { version = "0.1.0" }
 nonmax = { version = "0.5.5", features = ["serde"] }
+strum = "0.26"
```

Comment:
Why do you need `strum` as well as `strum_macros`, I don't see it being used.

Reply:
Will fix, sec